parm/config/gfs/config.wave

```diff
@@ -99,18 +99,18 @@ fi
 export WAVHINDH=${WAVHINDH:-0}
 export FHMIN_WAV=${FHMIN_WAV:-0}
 export FHOUT_WAV=${FHOUT_WAV:-3}
-export FHMAX_HF_WAV=${FHMAX_HF_WAV:-120}
+export FHMAX_HF_WAV=${FHMAX_HF_WAV:-0} # 120 for ops
 export FHOUT_HF_WAV=${FHOUT_HF_WAV:-1}
 export FHMAX_WAV_IBP=180
 if (( FHMAX_WAV < FHMAX_WAV_IBP )); then export FHMAX_WAV_IBP=${FHMAX_GFS} ; fi
 
 # gridded and point output rate
 export DTFLD_WAV=$(( FHOUT_HF_WAV * 3600 ))
-export DTPNT_WAV=${DTPNT_WAV:-3600}
+export DTPNT_WAV=${DTPNT_WAV:-10800} # 3600 for ops
 export FHINCP_WAV=$(( DTPNT_WAV / 3600 ))
 
 # Selected output parameters (gridded)
-export OUTPARS_WAV=${OUTPARS_WAV:-"WND HS FP DP PHS PTP PDIR"}
+export OUTPARS_WAV="WND CUR ICE HS T01 T02 DIR FP DP PHS PTP PDIR CHA"
```

Comment:
Can we at least make a comment here about what the ops settings for v16 were?  This is not critical, but would be nice.

Reply:
The v16 ops settings can always be referred to from one of the v16 tags on GitHub.  Retaining v16 information in develop for all components could lead to comment clutter.

Reply:
It would have been nice to have had a setting saying what the previous ops settings were as we used the defaults.s2s to overwrite those for development options.  But you are correct that they do indeed exist somewhere else, so this can be skipped.  That being said, comments can be very useful and "comment clutter" is very subjective to the person who finds the comment useful.